### ISSUE TITLE
Error in return from .status() function : UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'value' of undefined at e.<anonymous> (/home/user/bluelinky/node_modules/bluelinky/dist/index.js:1:9467)

### ISSUE THREAD

**Issue by deepcoder** [Hacksore/bluelinky#61]
Error in return from .status() function : UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'value' of undefined at e.<anonymous> (/home/user/bluelinky/node_modules/bluelinky/dist/index.js:1:9467)

I am trying the code shown below. All functions except .status() seem to be working. The .status() seems to return correct response from Bluelink, but something seems to be crashing in parsing the returned data. I have tried several different formats of arguments to the .status() call and all fail the same. Thanks in advance for your help! -- Dave

```javascript
#!/usr/bin/env node

const BlueLinky = require('bluelinky');

const client = new BlueLinky({
  username: 'x@y.com',
  password: '1234',
  region: 'US',
  pin: '1234'
});

var vehicle;
var status;

client.on('ready', async () => {

  console.log('getVehicle() ----------------------------------------------------------------------------');
  vehicle = client.getVehicle('XXXXXXXXXXXXXXXXX');
  status = await vehicle;
  console.log(status);

  console.log('location()   ----------------------------------------------------------------------------');
  status = await vehicle.location();
  console.log(status);

  console.log('status()     ----------------------------------------------------------------------------');
  status = await vehicle.status({ parsed: false, refresh: false });
  //status = await vehicle.status();
  console.log(status);


});
```

==================================

location()   ----------------------------------------------------------------------------
[2020-08-01 14:51:06] debug: Token is all good, moving on!
[2020-08-01 14:51:44] debug: {"head":178,"coord":{"alt":41.0,"lon":-123.123456,"type":0,"lat":123.123456},"accuracy":{"pdop":13,"hdop":7},"time":"2020-08-01T21:46:10Z","speed":{"unit":1,"value":0}}
{
  latitude: 123.123456,
  longitude: -123.123456,
  altitude: 41,
  speed: { unit: 1, value: 0 },
  heading: 178
}
status()     ----------------------------------------------------------------------------
[2020-08-01 14:51:44] debug: Token is all good, moving on!
[2020-08-01 14:51:52] debug: {"hataTID":"0zfFAuxxxx-rJjwlyy_6Xg","vehicleStatus":{"dateTime":"2020-08-01T20:57:58Z","acc":false,"trunkOpen":false,"doorLock":false,"defrostStatus":"false","transCond":true,"doorLockStatus":"false","doorOpen":{"frontRight":0,"frontLeft":0,"backLeft":0,"backRight":0},"airCtrlOn":false,"airTemp":{"unit":0,"hvacTempType":0,"value":"01H"},"evStatus":{"valueDiff":0,"remainTime2":{"etc3":{"unit":1,"value":0},"etc2":{"unit":1,"value":0},"atc":{"unit":1,"value":0},"etc1":{"unit":1,"value":0}},"evIgnitionStatus":true,"batteryPlugin":0,"timeDiff":1,"batteryCharge":false,"batteryStatus":99,"remainTime":[],"drvDistance":[{"rangeByFuel":{"gasModeRange":{"unit":3,"value":597.0},"totalAvailableRange":{"unit":3,"value":626.0},"evModeRange":{"unit":3,"value":29.0}},"type":2}]},"battery":{"batSignalReferenceValue":{},"batSoc":67,"batState":0,"sjbDeliveryMode":0},"ign3":true,"ignitionStatus":"false","lowFuelLight":false,"sideBackWindowHeat":0,"engine":false,"defrost":false,"hoodOpen":false,"airConditionStatus":"false","steerWheelHeat":0,"tirePressureLamp":{"tirePressureWarningLampRearLeft":0,"tirePressureWarningLampFrontLeft":0,"tirePressureWarningLampFrontRight":0,"tirePressureWarningLampAll":0,"tirePressureWarningLampRearRight":0},"trunkOpenStatus":"false"}}
(node:19974) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'value' of undefined
    at e.<anonymous> (/home/user/bluelinky/node_modules/bluelinky/dist/index.js:1:9467)
    at /home/user/bluelinky/node_modules/bluelinky/dist/index.js:1:2098
    at Object.next (/home/user/bluelinky/node_modules/bluelinky/dist/index.js:1:2203)
    at n (/home/user/bluelinky/node_modules/bluelinky/dist/index.js:1:936)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:19974) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:19974) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by Hacksore** [comment]
I imagine this is caused by the same issue with parsed status in #63 as it tries to create the parsed structure and fails.

I did just commit 556dc273f88bf21a031e176fa6b16cf167ead8e6 to develop which might mitigate this.

**Comment by Hacksore** [comment]
Should be fixed with #67
